skyreader/utils/handle_map_data.py

```diff
@@ -33,13 +37,15 @@ def extract_map(
         - grid_dec: declination for each pixel in grid_value
         - equatorial_map: healpix map with maximum nside (all pixels
             with same nside)
+        - uniq_array: uniqs for all the pixels of the map (optional)
     """
 
     grid_map = dict()
 
     nsides = result.nsides
     max_nside = max(nsides)
     equatorial_map = np.full(healpy.nside2npix(max_nside), np.nan)
+    uniq_list = list()
```

Comment:
Technically, `uniq_list = []` would be faster since it doesn't require a function call at runtime (the same goes for the other built-in types). I see you have these in many places. 

That said, it's a style choice.

Reply:
Done! Thanks, I didn't know about this